### DISCUSSION TITLE
求救大佬

### DISCUSSION THREAD

**Discussion by Lambda-D3L7A** [viopsa233/AI-testers-Forum#12] [Q&A]
![屏幕截图 2023-04-20 015526](https://user-images.githubusercontent.com/131350374/233316777-f83b1444-c6cb-4665-aa10-693ce5ffa4ff.png)
正在安装pandoraAI时爆error
如何解决这个问题

**Comment by viopsa233** [comment]
你安装了[node.js](https://nodejs.org/zh-tw/download)了没有
